integrations/compare_simplesearcher.py

```diff
@@ -0,0 +1,69 @@
+#
+# Pyserini: Python interface to the Anserini IR toolkit built on Lucene
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import filecmp
+import os
+from typing import List
+
+
+class CompareSimpleSearcher:
+    def __init__(self, index: str, topics: str):
+        self.index_path = index
+        self.topics = topics
+        self.pyserini_base_cmd = 'python3 -m pyserini.search'
+
+    @staticmethod
+    def _cleanup(files: List[str]):
+        for file in files:
+            if os.path.exists(file):
+                os.remove(file)
+
+    def run(self, runtag: str, extras_one: str, extras_two: List[str]):
```

Comment:
I'm not sure I like `extras_one` and `extras_two`...

How about this as a re-design:
- the simple harness takes a bunch of args, performs a run, and returns the MD5 checksum of the run.
- the actual test case takes the harness, submits two separate runs (with different threads and batch sizes), and compares the checksum; tests pass if the checksums match.
-